src/web/components/ClientSideTokenGeneration/CstgAddDomainDialog.tsx

```diff
@@ -46,27 +45,16 @@ function CstgAddDomainDialog({
         message: 'The domains entered already exist.',
       });
     } else {
-      const invalidDomains: string[] = [];
-      let allValid = true;
-      uniqueDomains.forEach((newDomain) => {
-        if (!isValidDomain(newDomain)) {
-          invalidDomains.push(newDomain);
-          allValid = false;
-        }
-      });
-      if (!allValid) {
+      // filter for uniqueness (e.g. 2 different domains entered could have the same root-level domain)
+      const dedupedDomains = deduplicateStrings(uniqueDomains);
```

Comment:
Could we not just do this in admin?

Reply:
In admin, duplicate domains will throw an error - but functionality-wise, on the front end it is not an error.  When the add domains dialog was being built, we discussed de-depuplicating automagically for the user if they enter duplicates and still have it be a successful add (unless every domain they enter already exists).  We could maybe do it in admin, it would just require admin code changes/approvals.  Lmk what you think

Reply:
New ticket will be created to fix this - not broken, just switching front end to back end